### ISSUE TITLE
Batch Size of Four

### ISSUE THREAD

**Issue by sagedemage** [sagedemage/pet_image_classification_system#11]
**Describe the enhancement**
Set the batch size of 4. A smaller batch size like 4 could help improve the speed of convergence and accuracy of the model.

For this modification to work, you must adjust the correct calculation of the model output in the `train_model.py` file starting at line 226: 
```
voutputs = model(vinputs)
pred_probab = torch.max(voutputs.data, 1)[1]
vloss = loss_fn(voutputs, vlabels)
running_test_loss += vloss
correct += (
    (pred_probab == vlabels)
    .type(torch.float)
    .sum()
    .item()
)
```

## Resource
- https://stackoverflow.com/questions/35050753/how-big-should-batch-size-and-number-of-epochs-be-when-fitting-a-model

**Comment by sagedemage** [comment]
```
git fetch origin
git checkout 11-batch-size-of-four
```
